pkg/outdated/registry.go

```diff
@@ -15,6 +16,13 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	// SemverOutlierMajorVersionThreshold defines the number of major versions that must be skipped before
+	// the next version is considered an outlier
+	// setting this to 1 allows NO major versions to be skipped
+	SemverOutlierMajorVersionThreshold = 1
```

Comment:
IMO this should default to two
I know there are legitimate repos that have skipped a major version before